pants-plugins/uses_services/BUILD

```diff
@@ -0,0 +1,16 @@
+python_sources(
+    sources=["*.py", "!*_test.py", "!data_fixtures.py"],
+)
+
+python_test_utils(
+    name="test_utils",
+    sources=["data_fixtures.py"],
+)
+
+python_tests(
+    name="tests",
+    overrides={
+        # We use this plugin to validate we can run the tests for it.
+        "mongo_rules_test.py": {"uses": ["mongo"]},
+    },
+)
```

Comment:
It's kind of circular, but I declared that the `is_mongo_running` test uses mongo. That means that the plugin is used before the plugin is tested. So, if the plugin is broken, we won't even get the test results.

I'm leaning towards removing or commenting this out. Thoughts?

Reply:
I don't like idea that if it fails we won't know why..

Reply:
Thanks for validating my suspicion. I've commented this out with a warning about why it should not be used.